scripts/reborn_webui_v2_live_qa/semantic_judge.py

```
@@ -0,0 +1,204 @@
+    message = choice.get("message")
+    if not isinstance(message, dict):
+        return ""
+    return message.get("content") or ""
+
+
+def _parse_json_object(text: object) -> object:
+    if not isinstance(text, str):
+        return None
+    try:
+        return json.loads(text)
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)